controller/sql-query-controller.js

```diff
@@ -6,4 +6,19 @@ exports.getAllTopics = async function(req, res)
     
     try { res.status(200).send({"topics": allTopics}) } 
     catch (error) { console.error(error); }
```

Comment:
The error needs to reach the client, so you'll need to make sure to respond to the client with an error. Console.error is an internal log.

Reply:
Next() is now used.